WORKSPACE

```diff
@@ -1,8 +1,18 @@
+workspace(name = "nes")
+
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
```

Comment:
You can load git_repository and new_git_repository on the same line. Just provide new_git_repository as an additional arg to the line above.